mantidproject/mantid
# Indirect - ElasticWindowMultiple doesn't propagate errors

**Description of work.**
This PR ensures that the ElasticWindowMultiple algorithm is propagating its errors during normalization. Previously, the errors seen in the _elf and _elt output workspaces where the same - this should not be the case as _elt is the normalized version of _elf.

**To test:**
1. `Interfaces`->`Indirect`->`Data Analysis`->`Elwin` interface
2. Load in all the data below all at once (by using `ctrl` when browsing for data)
3. Check the `Normalise to Lowest Temp` check box
4. Click run
5. Open the _elf and _elt workspaces, the errors should be different

**Data Files**
[elwin_test_data.zip](https://github.com/mantidproject/mantid/files/2768058/elwin_test_data.zip)

Fixes #21401 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
